docs-chef-io/content/inspec/waivers.md

```diff
@@ -58,3 +71,34 @@ xccdf_org.cisecurity.benchmarks_rule_1.1.1.4_Ensure_mounting_of_hfs_filesystems_
   justification: "This might be a bug in the test. @qateam"
   run: false
 ```
+
+Example in JSON:
+
+```json
+{
+  "waiver_control_1_2_3": {
+    "expiration_date": "2019-10-15T00:00:00.000Z",
+    "justification": "Not needed until Q3. @secteam"
+  },
+  "xccdf_org.cisecurity.benchmarks_rule_1.1.1.4_Ensure_mounting_of_hfs_filesystems_is_disabled": {
+    "expiration_date": "2020-03-01T00:00:00.000Z",
+    "justification": "This might be a bug in the test. @qateam",
+    "run": false
+  }
+}
+```
+
+Example in CSV:
+
+These file formats support the following fields in a file:
+
+* `control_id`
+  _Required_.
+* `justification`
+  _Required_.
+* `run`
+  _Optional_.
+* `expiration_date`
+  _Optional_.
+
+![Waiver File Example](/images/inspec/waivers_file_excel.png)
```

Comment:
Can you replace the image with just a MD table. We don't want exclude users who use screen readers.